src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -86,15 +85,42 @@ def handler(args, extra_args):
 def _add_model_args(parser):
     model_group = parser.add_argument_group("Model")
 
+    model_group.add_argument(
+        "--expected-output-tokens",
+        type=int,
+        default=LlmInputs.DEFAULT_EXPECTED_OUTPUT_TOKENS,
+        required=False,
+        help="The number of tokens to expect in the output. "
+        "This is used to determine the length of the prompt. "
```

Comment:
I don't believe this is used to determine the length of the prompt.

Reply:
Pulled from the docstring in the module. 
@nv-braf is there a more appropriate help message?

Reply:
not worth holding up freeze for. approving